server/pageRenderer.js

```diff
@@ -68,6 +68,10 @@ const readyHtml = (app) => {
   }
 };
 
+const analytics = __DEV__
+  ? ''
+  : `<script defer data-domain=${config.webUrl} src="https://analytics.webkom.dev/js/plausible.js"></script>`;
```

Comment:
`config.webUrl` includes `https://`, but does that matter?

Reply:
Fixed now. I added a quick regex replace for all URL protocols